README.md

```diff
@@ -1,6 +1,6 @@
 # Google CloudEvents Types for Java
 
-[![Maven Central](https://img.shields.io/maven-central/v/com.google.cloud/google-cloudevent-types.svg)](https://central.sonatype.com/artifact/com.google.cloud/google-cloudevent-types/0.8.0)
+[![Maven Central](https://img.shields.io/maven-central/v/com.google.cloud/google-cloudevent-types.svg)](https://central.sonatype.com/artifact/com.google.cloud/google-cloudevent-types/0.7.0)
```

Comment:
Intended rollback?

Reply:
Yes, rollouts to maven central are slow so this could be a 404 or 500 error for a couple hours. I have issue https://github.com/googleapis/google-cloudevents-java/issues/157 open to address further.